arcgis-ios-sdk-samples/Display information/Custom dictionary style/README.md

```diff
@@ -0,0 +1,36 @@
+# Custom dictionary style
+
+Use a custom dictionary style (.stylx) to symbolize features using a variety of attribute values.
+
+![Image of custom dictionary style](image1.png)
```

Comment:
Should we leave this image not renamed? This sample was created before the README updates started.